packages/vite/src/esbuild-resolver.ts

```diff
@@ -42,7 +42,8 @@ export function esBuildResolver(): EsBuildPlugin {
     if (path.endsWith('.hbs')) {
       src = hbsToJS(src);
     } else if (['.gjs', '.gts'].some(ext => path.endsWith(ext))) {
-      src = preprocessor.process(src, { filename: path });
+      let { code /*,  map */ } = preprocessor.process(src, { filename: path });
+      src = code;
```

Comment:
I guess for this case we would actually need inline source map?

Reply:
probably, ya. added!

Reply:
I mean it probably won't make much difference here since this is mostly just used for dependency discovery in the esbuild phase 🤔

Reply:
Oh, right. Addons already pre compiled the gjs